src/utils/util.jl

```diff
@@ -183,15 +183,14 @@ return Vector{Vector{Int64}}.
 function get_activation(nnet::Network, bounds::Vector{Hyperrectangle})
     act_pattern = Vector{Vector{Int}}(undef, length(nnet.layers))
     for (i, layer) in enumerate(nnet.layers)
-        act_pattern[i] = get_activation(layer, bounds[i])
+        act_pattern[i] = get_activation(layer, bounds[i+1])
     end
     return act_pattern
 end
 
 function get_activation(L::Layer{ReLU}, bounds::Hyperrectangle)
-    before_act_bound = approximate_affine_map(L, bounds)
-    lower = low(before_act_bound)
-    upper = high(before_act_bound)
+    lower = low(bounds)
+    upper = high(bounds)
```

Comment:
This is a dangerous change... this function is used in lots of places. You're sure it doesn't have ill effects?

Also lets update the docstring and maybe the variable name to reflect these are pre-activation bounds.

Reply:
I believe it should be equivalent to what we had before? Before we had the approximate affine map happening in bounds. Now I was trying to consistently use pre-activation bounds, and so I used them here as well but I think it effectively is just switching where approximate affine map is getting applied? Unless I'm misinterpreting this? Could it be a different approximate affine map that gets used? Although it was already being used on bounds, just on post-activation bounds, so we're still dealing with approximate affine maps on hyperrectangles.

Reply:
I think if it is equivalent it might be cleaner this way, since we don't have to do the approximate affine map again ( in the old version it's already being done in get_bounds when we get post-activation bounds, then we apply it again in this function).

Reply:
The only place I found where get_bounds is called with bounds passed in is in Planet. Since I updated Planet to use pre-activation bounds,  this should be okay. All other uses of get_activation seem to pass in a single point, so I think those should still be ok.

Reply:
I updated the doc string - in general I don't know what to do between pre and post activation bounds.